app/src/main/java/zechs/drive/stream/data/model/DriveFile.kt

```diff
@@ -5,11 +5,12 @@ import zechs.drive.stream.utils.util.Converter
 
 @Keep
 data class DriveFile(
-    val id: String,
+    var id: String,
```

Comment:
I do not understand the purpose of making it non-final.

I can see its used to handle `id` for shortcuts for which you should rather use Kotlin [copy()](https://kotlinlang.org/docs/data-classes.html#copying) function and re-assign the `id` something like `file.copy(id=shortcut.id)`